src/main/java/com/fasterxml/jackson/databind/node/ObjectNode.java

```diff
@@ -87,7 +87,8 @@ public ObjectNode with(String exprOrProperty) {
                 .getClass().getName() + "`)");
         }
         ObjectNode result = objectNode();
-        return _put(exprOrProperty, result);
+        _put(exprOrProperty, result);
+        return result;
```

Comment:
What does _put return to create regression ? Seems like it should work.

Reply:
I looked at original change and I think it's the difference between returning `this` (which `_put()` returns vs `ObjectNode` being created that is to be returned here.

But I'd appreciate it if you can double-check @JooHyukKim .